x-pack/plugins/xpack_main/server/routes/api/v1/features/features.ts

```diff
@@ -10,6 +10,9 @@ export function featuresRoute(server: Record<string, any>) {
   server.route({
     path: '/api/features/v1',
     method: 'GET',
+    options: {
+      tags: ['access:manage_feature_controls'],
```

Comment:
```suggestion
      tags: ['access:features'],
```

We're currently only using this API in the context of "managing feature controls", but there's potential we'll want to consume this elsewhere, hence the suggestion.

Reply:
> We're currently only using this API in the context of "managing feature controls", but there's potential we'll want to consume this elsewhere, hence the suggestion.

I opted for the more specific tag name because we don't scope/namespace these by feature, so it's possible for two features to define the same access tag. That could unintentionally grant a user access to an API that they shouldn't have access to. 
It is rather unlikely that another feature would register a tag called "features", so I can change this if you don't have concerns around this.

Reply:
> It is rather unlikely that another feature would register a tag called "features", so I can change this if you don't have concerns around this.

It's a fair point... I still feel like "manage_feature_controls" is just the wrong tag for it though, but I don't want to be ignoring your concern for tag collisions. It feels like the point you've brought up is something that warrants more consideration. 

How would you feel about renaming this to `features` and creating an issue for us to discuss other solutions to potentially prevent unintentional collisions. Perhaps the changes we're making to port this to the new platform will provide us more opportunities than the "tag" based approach which we're employing because of Hapi.

Reply:
That sounds good to me, I'll make it so